app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/optimizelyCommunicatorFactory.ts

```diff
@@ -0,0 +1,17 @@
+import { Communicator } from '../../types/Communicator'
+import { UserPreferences } from '../../types/UserPreferences'
+
+const optimizelyCommunicatorFactory = (window: Window): Communicator => ({
+  sendPreferences: (userPreferences: UserPreferences) => {
+
+    const setOptimizelyOptOut = (optOut: Boolean) => {
+      window.optimizely.push({ 'type': 'optOut', 'isOptOut': optOut });
```

Comment:
You could change the optOut param to isOptOut, then simplify as
window.optimizely.push({ 'type': 'optOut', isOptOut });

Reply:
Thanks, didn't know that, good learning.